pkg/packagekit/wix/service.go

```diff
@@ -142,6 +142,15 @@ func WithDelayedStart() ServiceOpt {
 	}
 }
 
+func WithDisabledService() ServiceOpt {
+	return func(s *Service) {
+		s.serviceInstall.Start = "disabled"
+		// If this is not set to empty the installer hangs trying to start the
+		// disabled service.
+		s.serviceControl.Start = ""
```

Comment:
Huh. Apparently I didn't define a constant for `""`. Do you think we should define the constant, or leave this?

Reply:
I'll make a constant.